kimchi/src/circuits/polynomials/rot.rs

```diff
@@ -333,12 +310,28 @@ fn create_witness_rot<F: PrimeField>(witness: &mut [Vec<F>; COLUMNS], word: u64,
     // Right input of the "FFAdd" for the bound equation
```

Comment:
I'm not sure you should refer to this as `FFAdd` as it makes it sound like a`FFAdd` gate, instead maybe just say "addition"

Reply:
Maybe I'm being paranoid, but above on line 308: `let rotated = shifted + excess;`
We are adding to `u64` here-- can you remind me what prevents an overflow?